content/en/docs/rules/conditions.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Condition Syntax
+weight: 3
+---
+
+A condition is a boolean expression related to a single event that has been detected by Falco. You can use fields related to every supported event, but we'll focus on syscalls as they're currently the most common. The language supports boolean operators and parentheses as you'd expect. For example a condition like:
```

Comment:
A condition is a boolean expression related to a single event that has been detected by Falco. You can use fields related to every supported event, but this document focuses on syscalls as they're currently the most common. The language supports boolean operators and parentheses as you'd expect. For example a condition like: